api/libproject/devstart.rb

```diff
@@ -2487,10 +2487,6 @@ def get_auth_domain(project)
   return get_fc_config(project)["registeredDomainName"]
 end
 
-def get_auth_domain_group(project)
```

Comment:
This is not used.  See similar code which is still used at `api/db-cdr/generate-cdr/libproject/devstart.rb`